osu.Game/Screens/Backgrounds/BackgroundScreenDefault.cs

```diff
@@ -62,7 +63,13 @@ private void load(IAPIProvider api, SkinManager skinManager, OsuConfigManager co
             Next();
 
             // helper function required for AddOnce usage.
-            void loadNextIfRequired() => Next();
+            void loadNextIfRequired()
+            {
+                if (!IsLoaded)
+                    return;
+
+                Next();
+            }
```

Comment:
Sounds like the value change callbacks should just be moved to `LoadComplete` instead? To ensure the screen is loaded before queuing background change.